projects/components/src/utils/test/datagrid/datagrid.wo.ts

```diff
@@ -220,26 +190,17 @@ export class ClrDatagridWidgetObject<T> extends BaseWidgetObject<T> {
      * @param col 0-based index of header
      */
     getColumnHeader(col: number): T {
-        // return this.locatorDriver.get(`${Css.COLUMN_TITLE}:nth-of-type(${col + 1})`).unwrap();
         return this.locatorForCssSelectors(`${Css.COLUMN_TITLE}:nth-of-type(${col + 1})`)();
     }
 
-    /**
-     * Returns a row button
-     * @param row 0-based index
-     */
-    getRowButtonContainer(row: number): T {
-        return this._getRow(row).get(Css.ROW_BUTTON_CONTAINER).unwrap();
-    }
-
     getSingleSelectionRadioLabel(row: number): T {
```

Comment:
This method was added with the intention that it would be used for selecting. Can't you just use this for your tests? Maybe it can be renamed?